packages/session/package.json

```diff
@@ -11,14 +11,13 @@
   "scripts": {
     "build": "tsc",
     "dev": "tsc --watch --preserveWatchOutput",
-    "test": "nyc --reporter=html --reporter=text mocha --watch-files src/ src/**/*.test.ts"
+    "test": "c8 --reporter=html --reporter=text --reporter=cobertura mocha --watch-files src/ src/**/*.test.ts"
```

Comment:
`nyc` wasn't able to work correctly with `tsx`; `c8` was! And since it uses native Node functionality, it should in theory perform much better.

Reply:
Nice!